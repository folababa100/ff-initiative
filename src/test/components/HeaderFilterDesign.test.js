import React from 'react';
import HeaderFilterDesign from '../../components/HeaderFIlterDesign';
import { shallow } from "enzyme";

test('should render component correctly', () => {
  const wrapper = shallow(<HeaderFilterDesign/>);
  expect(wrapper).toMatchSnapshot()
})
