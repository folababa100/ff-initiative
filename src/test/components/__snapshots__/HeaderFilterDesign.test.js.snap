// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HeaderFilterDesign />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <nav
        className="navbar navbar-adjust-all navbar-expand-lg navbar-light"
>
        <div
                className="container"
        >
                <NavLink
                        activeClassName="active"
                        aria-current="page"
                        className="aa navbar-brand brand-adjust"
                        to="/"
                >
                        <img
                                alt="FF-Initiative Logo"
                                className="d-inline-block align-top"
                                height="70"
                                src="/images/fola-logo.png"
                                width="50"
                        />
                        FF-Initiative
                </NavLink>
                <form
                        className="mr-4 flex reset-search"
                >
                        <input
                                className="reset-form reset-form-radius"
                                placeholder="Search Products"
                                type="search"
                        />
                        <button
                                className="btn reset-btn"
                        >
                                <i
                                        className="ion-ios-search"
                                />
                        </button>
                </form>
                <ul
                        className="navbar-nav"
                >
                        <li
                                className="nav-item flex"
                        >
                                <i
                                        className="ion-android-favorite-outline x3"
                                />
                                <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="nav-link line-adjust"
                                        to="/like"
                                >
                                        Liked Items
                                </NavLink>
                        </li>
                        <li
                                className="nav-item flex"
                        >
                                <i
                                        className="ion-ios-cart-outline x3"
                                />
                                <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="nav-link line-adjust"
                                        to="/cart"
                                >
                                        Cart
                                </NavLink>
                        </li>
                </ul>
        </div>
</nav>,
      "className": "white-color",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    className="aa navbar-brand brand-adjust"
                    to="/"
          >
                    <img
                              alt="FF-Initiative Logo"
                              className="d-inline-block align-top"
                              height="70"
                              src="/images/fola-logo.png"
                              width="50"
                    />
                    FF-Initiative
          </NavLink>
          <form
                    className="mr-4 flex reset-search"
          >
                    <input
                              className="reset-form reset-form-radius"
                              placeholder="Search Products"
                              type="search"
                    />
                    <button
                              className="btn reset-btn"
                    >
                              <i
                                        className="ion-ios-search"
                              />
                    </button>
          </form>
          <ul
                    className="navbar-nav"
          >
                    <li
                              className="nav-item flex"
                    >
                              <i
                                        className="ion-android-favorite-outline x3"
                              />
                              <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="nav-link line-adjust"
                                        to="/like"
                              >
                                        Liked Items
                              </NavLink>
                    </li>
                    <li
                              className="nav-item flex"
                    >
                              <i
                                        className="ion-ios-cart-outline x3"
                              />
                              <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="nav-link line-adjust"
                                        to="/cart"
                              >
                                        Cart
                              </NavLink>
                    </li>
          </ul>
</div>,
        "className": "navbar navbar-adjust-all navbar-expand-lg navbar-light",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              aria-current="page"
              className="aa navbar-brand brand-adjust"
              to="/"
>
              <img
                            alt="FF-Initiative Logo"
                            className="d-inline-block align-top"
                            height="70"
                            src="/images/fola-logo.png"
                            width="50"
              />
              FF-Initiative
</NavLink>,
            <form
              className="mr-4 flex reset-search"
>
              <input
                            className="reset-form reset-form-radius"
                            placeholder="Search Products"
                            type="search"
              />
              <button
                            className="btn reset-btn"
              >
                            <i
                                          className="ion-ios-search"
                            />
              </button>
</form>,
            <ul
              className="navbar-nav"
>
              <li
                            className="nav-item flex"
              >
                            <i
                                          className="ion-android-favorite-outline x3"
                            />
                            <NavLink
                                          activeClassName="active"
                                          aria-current="page"
                                          className="nav-link line-adjust"
                                          to="/like"
                            >
                                          Liked Items
                            </NavLink>
              </li>
              <li
                            className="nav-item flex"
              >
                            <i
                                          className="ion-ios-cart-outline x3"
                            />
                            <NavLink
                                          activeClassName="active"
                                          aria-current="page"
                                          className="nav-link line-adjust"
                                          to="/cart"
                            >
                                          Cart
                            </NavLink>
              </li>
</ul>,
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": Array [
                <img
                  alt="FF-Initiative Logo"
                  className="d-inline-block align-top"
                  height="70"
                  src="/images/fola-logo.png"
                  width="50"
/>,
                "FF-Initiative",
              ],
              "className": "aa navbar-brand brand-adjust",
              "to": "/",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "FF-Initiative Logo",
                  "className": "d-inline-block align-top",
                  "height": "70",
                  "src": "/images/fola-logo.png",
                  "width": "50",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "FF-Initiative",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <input
                  className="reset-form reset-form-radius"
                  placeholder="Search Products"
                  type="search"
/>,
                <button
                  className="btn reset-btn"
>
                  <i
                                    className="ion-ios-search"
                  />
</button>,
              ],
              "className": "mr-4 flex reset-search",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "reset-form reset-form-radius",
                  "placeholder": "Search Products",
                  "type": "search",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <i
                    className="ion-ios-search"
/>,
                  "className": "btn reset-btn",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "ion-ios-search",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                "type": "button",
              },
            ],
            "type": "form",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="nav-item flex"
>
                  <i
                                    className="ion-android-favorite-outline x3"
                  />
                  <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    className="nav-link line-adjust"
                                    to="/like"
                  >
                                    Liked Items
                  </NavLink>
</li>,
                <li
                  className="nav-item flex"
>
                  <i
                                    className="ion-ios-cart-outline x3"
                  />
                  <NavLink
                                    activeClassName="active"
                                    aria-current="page"
                                    className="nav-link line-adjust"
                                    to="/cart"
                  >
                                    Cart
                  </NavLink>
</li>,
              ],
              "className": "navbar-nav",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <i
                      className="ion-android-favorite-outline x3"
/>,
                    <NavLink
                      activeClassName="active"
                      aria-current="page"
                      className="nav-link line-adjust"
                      to="/like"
>
                      Liked Items
</NavLink>,
                  ],
                  "className": "nav-item flex",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "ion-android-favorite-outline x3",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "activeClassName": "active",
                      "aria-current": "page",
                      "children": "Liked Items",
                      "className": "nav-link line-adjust",
                      "to": "/like",
                    },
                    "ref": null,
                    "rendered": "Liked Items",
                    "type": [Function],
                  },
                ],
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <i
                      className="ion-ios-cart-outline x3"
/>,
                    <NavLink
                      activeClassName="active"
                      aria-current="page"
                      className="nav-link line-adjust"
                      to="/cart"
>
                      Cart
</NavLink>,
                  ],
                  "className": "nav-item flex",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "ion-ios-cart-outline x3",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "activeClassName": "active",
                      "aria-current": "page",
                      "children": "Cart",
                      "className": "nav-link line-adjust",
                      "to": "/cart",
                    },
                    "ref": null,
                    "rendered": "Cart",
                    "type": [Function],
                  },
                ],
                "type": "li",
              },
            ],
            "type": "ul",
          },
        ],
        "type": "div",
      },
      "type": "nav",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <nav
          className="navbar navbar-adjust-all navbar-expand-lg navbar-light"
>
          <div
                    className="container"
          >
                    <NavLink
                              activeClassName="active"
                              aria-current="page"
                              className="aa navbar-brand brand-adjust"
                              to="/"
                    >
                              <img
                                        alt="FF-Initiative Logo"
                                        className="d-inline-block align-top"
                                        height="70"
                                        src="/images/fola-logo.png"
                                        width="50"
                              />
                              FF-Initiative
                    </NavLink>
                    <form
                              className="mr-4 flex reset-search"
                    >
                              <input
                                        className="reset-form reset-form-radius"
                                        placeholder="Search Products"
                                        type="search"
                              />
                              <button
                                        className="btn reset-btn"
                              >
                                        <i
                                                  className="ion-ios-search"
                                        />
                              </button>
                    </form>
                    <ul
                              className="navbar-nav"
                    >
                              <li
                                        className="nav-item flex"
                              >
                                        <i
                                                  className="ion-android-favorite-outline x3"
                                        />
                                        <NavLink
                                                  activeClassName="active"
                                                  aria-current="page"
                                                  className="nav-link line-adjust"
                                                  to="/like"
                                        >
                                                  Liked Items
                                        </NavLink>
                              </li>
                              <li
                                        className="nav-item flex"
                              >
                                        <i
                                                  className="ion-ios-cart-outline x3"
                                        />
                                        <NavLink
                                                  activeClassName="active"
                                                  aria-current="page"
                                                  className="nav-link line-adjust"
                                                  to="/cart"
                                        >
                                                  Cart
                                        </NavLink>
                              </li>
                    </ul>
          </div>
</nav>,
        "className": "white-color",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
>
            <NavLink
                        activeClassName="active"
                        aria-current="page"
                        className="aa navbar-brand brand-adjust"
                        to="/"
            >
                        <img
                                    alt="FF-Initiative Logo"
                                    className="d-inline-block align-top"
                                    height="70"
                                    src="/images/fola-logo.png"
                                    width="50"
                        />
                        FF-Initiative
            </NavLink>
            <form
                        className="mr-4 flex reset-search"
            >
                        <input
                                    className="reset-form reset-form-radius"
                                    placeholder="Search Products"
                                    type="search"
                        />
                        <button
                                    className="btn reset-btn"
                        >
                                    <i
                                                className="ion-ios-search"
                                    />
                        </button>
            </form>
            <ul
                        className="navbar-nav"
            >
                        <li
                                    className="nav-item flex"
                        >
                                    <i
                                                className="ion-android-favorite-outline x3"
                                    />
                                    <NavLink
                                                activeClassName="active"
                                                aria-current="page"
                                                className="nav-link line-adjust"
                                                to="/like"
                                    >
                                                Liked Items
                                    </NavLink>
                        </li>
                        <li
                                    className="nav-item flex"
                        >
                                    <i
                                                className="ion-ios-cart-outline x3"
                                    />
                                    <NavLink
                                                activeClassName="active"
                                                aria-current="page"
                                                className="nav-link line-adjust"
                                                to="/cart"
                                    >
                                                Cart
                                    </NavLink>
                        </li>
            </ul>
</div>,
          "className": "navbar navbar-adjust-all navbar-expand-lg navbar-light",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                aria-current="page"
                className="aa navbar-brand brand-adjust"
                to="/"
>
                <img
                                alt="FF-Initiative Logo"
                                className="d-inline-block align-top"
                                height="70"
                                src="/images/fola-logo.png"
                                width="50"
                />
                FF-Initiative
</NavLink>,
              <form
                className="mr-4 flex reset-search"
>
                <input
                                className="reset-form reset-form-radius"
                                placeholder="Search Products"
                                type="search"
                />
                <button
                                className="btn reset-btn"
                >
                                <i
                                                className="ion-ios-search"
                                />
                </button>
</form>,
              <ul
                className="navbar-nav"
>
                <li
                                className="nav-item flex"
                >
                                <i
                                                className="ion-android-favorite-outline x3"
                                />
                                <NavLink
                                                activeClassName="active"
                                                aria-current="page"
                                                className="nav-link line-adjust"
                                                to="/like"
                                >
                                                Liked Items
                                </NavLink>
                </li>
                <li
                                className="nav-item flex"
                >
                                <i
                                                className="ion-ios-cart-outline x3"
                                />
                                <NavLink
                                                activeClassName="active"
                                                aria-current="page"
                                                className="nav-link line-adjust"
                                                to="/cart"
                                >
                                                Cart
                                </NavLink>
                </li>
</ul>,
            ],
            "className": "container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": Array [
                  <img
                    alt="FF-Initiative Logo"
                    className="d-inline-block align-top"
                    height="70"
                    src="/images/fola-logo.png"
                    width="50"
/>,
                  "FF-Initiative",
                ],
                "className": "aa navbar-brand brand-adjust",
                "to": "/",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "FF-Initiative Logo",
                    "className": "d-inline-block align-top",
                    "height": "70",
                    "src": "/images/fola-logo.png",
                    "width": "50",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "FF-Initiative",
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <input
                    className="reset-form reset-form-radius"
                    placeholder="Search Products"
                    type="search"
/>,
                  <button
                    className="btn reset-btn"
>
                    <i
                                        className="ion-ios-search"
                    />
</button>,
                ],
                "className": "mr-4 flex reset-search",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "reset-form reset-form-radius",
                    "placeholder": "Search Products",
                    "type": "search",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "input",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <i
                      className="ion-ios-search"
/>,
                    "className": "btn reset-btn",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "ion-ios-search",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "i",
                  },
                  "type": "button",
                },
              ],
              "type": "form",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="nav-item flex"
>
                    <i
                                        className="ion-android-favorite-outline x3"
                    />
                    <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="nav-link line-adjust"
                                        to="/like"
                    >
                                        Liked Items
                    </NavLink>
</li>,
                  <li
                    className="nav-item flex"
>
                    <i
                                        className="ion-ios-cart-outline x3"
                    />
                    <NavLink
                                        activeClassName="active"
                                        aria-current="page"
                                        className="nav-link line-adjust"
                                        to="/cart"
                    >
                                        Cart
                    </NavLink>
</li>,
                ],
                "className": "navbar-nav",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <i
                        className="ion-android-favorite-outline x3"
/>,
                      <NavLink
                        activeClassName="active"
                        aria-current="page"
                        className="nav-link line-adjust"
                        to="/like"
>
                        Liked Items
</NavLink>,
                    ],
                    "className": "nav-item flex",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "className": "ion-android-favorite-outline x3",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "function",
                      "props": Object {
                        "activeClassName": "active",
                        "aria-current": "page",
                        "children": "Liked Items",
                        "className": "nav-link line-adjust",
                        "to": "/like",
                      },
                      "ref": null,
                      "rendered": "Liked Items",
                      "type": [Function],
                    },
                  ],
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <i
                        className="ion-ios-cart-outline x3"
/>,
                      <NavLink
                        activeClassName="active"
                        aria-current="page"
                        className="nav-link line-adjust"
                        to="/cart"
>
                        Cart
</NavLink>,
                    ],
                    "className": "nav-item flex",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "className": "ion-ios-cart-outline x3",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "i",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "function",
                      "props": Object {
                        "activeClassName": "active",
                        "aria-current": "page",
                        "children": "Cart",
                        "className": "nav-link line-adjust",
                        "to": "/cart",
                      },
                      "ref": null,
                      "rendered": "Cart",
                      "type": [Function],
                    },
                  ],
                  "type": "li",
                },
              ],
              "type": "ul",
            },
          ],
          "type": "div",
        },
        "type": "nav",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
